sfcasts/chain-of-responsibility.md

```diff
@@ -57,9 +59,10 @@ pattern as part of the interface name so it's more obvious what pattern we're
 using.
 
 Now we can add the first method - `public function handle()` - and the
-arguments - `Character $player` and `FightResult $fightResult`. We're
-using `FightResult` here because we're only interested in the data of
-this `$player` object. Oh! And don't forget to add the return type `int`.
+arguments - `Character $player` and `FightResult $fightResult`. These
```

Comment:
@MolloKhan, I was a bit confused by this sentence. Let me know if this sounds ok.

Reply:
Yes it sounds good to me. The thing is we have 2 classes `FightResultSet` and `FightResult`, the former is like a wrapper of `FightResult` it holds one of those per each `Character` object. I just wanted to clarify which one we're using, but I think I achieved the opposite 😅